pkg/oci/client/load_balancer.go

```diff
@@ -150,18 +150,19 @@ func (c *client) GetCertificateByName(ctx context.Context, lbID, name string) (*
 	return nil, errors.WithStack(errNotFound)
 }
 
-func (c *client) CreateCertificate(ctx context.Context, lbID, certificate, key string) (string, error) {
+func (c *client) CreateCertificate(ctx context.Context, lbID string, cert loadbalancer.CertificateDetails) (string, error) {
 	if !c.rateLimiter.Writer.TryAccept() {
 		return "", RateLimitError(true, "CreateCertificate")
 	}
 
-	// TODO(apryde): We currently don't have a mechanism for supplying
-	// CreateCertificateDetails.CaCertificate.
 	resp, err := c.loadbalancer.CreateCertificate(ctx, loadbalancer.CreateCertificateRequest{
 		LoadBalancerId: &lbID,
 		CreateCertificateDetails: loadbalancer.CreateCertificateDetails{
```

Comment:
Could you not just do `CreateCertificateDetails: cert,` here?

Reply:
No, one is CreateCertificateDetails type and one is CertificateDetails. Unless there's some magic to make that happen? :)

Reply:
please say there is to make go cooler for me

Reply:
:man_facepalming:

Reply:
Goodness no! Facepalm that I didn't notice they were different types :smile: